CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.7.3
+
+This release adds 5 new rules along with some minor bug fixes and other improvements.
+
+### Rule changes
+
+- Add rule for `missing-end-label` ([#407](https://github.com/PlasmaFAIR/fortitude/pull/407))
+- Add rule `single-quote-string` ([#423](https://github.com/PlasmaFAIR/fortitude/pull/423))
+- Add rule for `misleading-inline-if-semicolon` ([#426](https://github.com/PlasmaFAIR/fortitude/pull/426))
+- Add rule `misleading-inline-if-continuation` ([#428](https://github.com/PlasmaFAIR/fortitude/pull/428))
```

Comment:
Just for consitency:

```suggestion
- Add rule `missing-end-label` ([#407](https://github.com/PlasmaFAIR/fortitude/pull/407))
- Add rule `single-quote-string` ([#423](https://github.com/PlasmaFAIR/fortitude/pull/423))
- Add rule `misleading-inline-if-semicolon` ([#426](https://github.com/PlasmaFAIR/fortitude/pull/426))
- Add rule `misleading-inline-if-continuation` ([#428](https://github.com/PlasmaFAIR/fortitude/pull/428))
```

Reply:
Whoops, that had been my intention, thanks for the fix!